contrib/sarif.tpl

```diff
@@ -1,12 +1,14 @@
 {
-  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.4.json",
+  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
```

Comment:
What has changed between these two versions? Just for my own curiosity how did you know to pick the new one?

Reply:
The previous one was a previous working draft. 
The SARIF format is now pretty stable. The specification **fix** it to this value.
Reference: [https://docs.oasis-open.org/sarif/sarif/v2.1.0/cs01/sarif-v2.1.0-cs01.html#_Toc16012437](https://docs.oasis-open.org/sarif/sarif/v2.1.0/cs01/sarif-v2.1.0-cs01.html#_Toc16012437)